### PR TITLE
Minor Modification and Reference Link addition

### PR DESCRIPTION
**Pull request by joinimran**
Added a  reference link to get details of the values required to set.

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/801

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365-atp.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Anti-spam, anti-malware, and anti-phishing are features of EOP that can be confi
 |Phishing email detection action|Quarantine message|Quarantine message||
 |High confidence Phish email detection action|Quarantine message|Quarantine message||
 |Bulk email detection action|Move message to Junk Email folder|Quarantine message||
-|Set Bulk email threshold to|6|4|The default is currently 7 but we recommend that most organizations mvoe it down to at least 6|
+|Set Bulk email threshold to|6|4|The default is currently 7 but it is recommended to make it 6. For details see [Bulk Complaint Level values](https://docs.microsoft.com/microsoft-365/security/office-365-security/bulk-complaint-level-values). |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/recommended-settings-for-eop-and-office365-atp.md]:
```suggestion
|Set Bulk email threshold to|6|4|The default is currently 7 but it is recommended to change it to 6. For details, see [Bulk Complaint Level values](https://docs.microsoft.com/microsoft-365/security/office-365-security/bulk-complaint-level-values). |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by joinimran** [comment]
Ready to go live

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #801.

Thank you.

cc: @kenwith

**Pull request merged**